test/maps/all_tribes.wmf/scripting/init.lua

```diff
@@ -29,7 +29,7 @@ function verify_buildings(playernumber, total_expected_buildings)
    for idx, building in ipairs(player.tribe.buildings) do
       -- We expect at least 1 building placed of each type.
       -- We can't create dismantlesites or check for constructionsites yet, so we skip them for the check.
-      if building.name ~= "constructionsite" and building.name ~= "dismantlesite" then
+      if building.name ~= "constructionsite" and building.name ~= "dismantlesite" and building.name ~= "amazons_treetop_sentry" then
```

Comment:
Please add explanation for treetop sentry to the comment too.
Maybe with a TODO line to revise if placing it gets implemented.